KachnaOnline.App/Controllers/EventsController.cs

```diff
@@ -0,0 +1,237 @@
+// EventsController.cs
+// Author: David Chocholatý
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using KachnaOnline.Business.Constants;
+using KachnaOnline.Business.Exceptions;
+using KachnaOnline.Business.Exceptions.Events;
+using KachnaOnline.Business.Facades;
+using KachnaOnline.Dto.Events;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace KachnaOnline.App.Controllers
+{
+    [ApiController]
+    [Route("events")]
+    [Authorize(Roles = RoleConstants.EventsManager)]
+    public class EventsController : ControllerBase
+    {
+        private readonly EventsFacade _eventsFacade;
+
+        public EventsController(EventsFacade eventsFacade)
+        {
+            _eventsFacade = eventsFacade;
+        }
+
+        /// <summary>
+        /// Returns the list of events being held at the specified time.
+        /// </summary>
+        /// <param name="at">Specifies the date and time when the events have to be held.</param>
```

Comment:
This is probably more of a system-wide issue. Looking at swagger, it's not clear how to actually input the date, what's the expected format, etc. This should, however, be addressed globally, I'd assume. CC @ondryaso

Reply:
The same format is to be used for states as well as board games. I believe it would be better to specify this once than for every single method where `DateTime` is used (either in query or in query body).